example/elm.json

```diff
@@ -1,12 +1,10 @@
 {
     "type": "application",
     "source-directories": [
-        ".",
```

Comment:
the elm compiler doesn't want nested source directories. So only include top-level directories. `../src/Multiselect` doesn't need to be included because the namespacing implies the directory nested under `../src`